alphagov/paas-rubbernecker
# Redefine stickers

## What

A little refactor of the current stickers system. Previously, as there were few, we had to create a function, then place multiple if statements around some HTML syntax for each sticker.

With the new approach, we should be able to add some details, into the JS Object and be done.

## How to review

- Code review
- Run the application
- Add some labels to story on Pivotal Tracker (`'small' task` preferably)
- See if the sticker appears
- Revert story to its previous state
